Automattic/wp-calypso
# Disable Native Login on the Desktop app for now

Native login hasn't been fully tested in the Desktop app yet so we're disabling it until we have more time to work on it.

### Testing Instructions
- From `wp-desktop`
- `cd calypso`
- `git pull -r`
- `git checkout disable/desktop-native-login`
- `cd ..`
- `make run`
- Check that oauth login and logout work like before

### Reviews

cc @v18
